tests/components/plugwise/test_init.py

```
@@ -137,13 +136,10 @@ async def test_coordinator_connect_exceptions(
     """Ensure _connect raises translated errors."""
     mock_smile_anna.connect.side_effect = side_effect
-    coordinator = PlugwiseDataUpdateCoordinator(
-        hass,
-        cooldown=0,
-        config_entry=mock_config_entry,
-    )
     with pytest.raises(expected_raise):
-        await coordinator._connect()
+        mock_config_entry.add_to_hass(hass)
+        await hass.config_entries.async_setup(mock_config_entry.entry_id)
+        await hass.async_block_till_done()
```

Ruff (0.14.8)

`pytest.raises()` block should contain a single simple statement

(PT012)